packages/components/Menu/src/MenuPopover/useMenuPopover.ts

```diff
@@ -23,7 +23,7 @@ export const useMenuPopover = (props: MenuPopoverProps): MenuPopoverState => {
 
   const { onKeyDown: onKeyDownProp, onKeyUp: onKeyUpProp } = props;
 
-  const onDismiss = React.useCallback(() => setOpen(undefined, false /* isOpen */), [setOpen]);
+  const onDismiss = props.onDismiss ?? React.useCallback(() => setOpen(undefined, false /* isOpen */), [setOpen]);
```

Comment:
Sorry, I meant that `onDismiss` should call `props.onDismiss`. I imagine we always need to call `setOpen` so it would be bad if the user providing a callback broke that?

Reply:
I thought user overriding onDismiss implied that the setOpen logic would be handled in the override. But then I see CM does call props.onDismiss on top of that so let's follow suit